src/api/webhooks.js

```diff
@@ -96,8 +87,6 @@ export class WebhooksApi extends BaseApi {
      */
     delete(webhookId) {
         const url = this.getFullURL('webhooks', webhookId);
-        return this.validateAuthHeaders().then((authHeaders) => {
-            return http('DELETE', url, undefined, authHeaders);
-        });
+        return this.request('DELETE', url);
```

Comment:
yeah I don't see why you should need it, I'd try smooch core with a script just to make sure